pkg/scheduler/api/devices/nvidia/vgpu/utils.go

```diff
@@ -100,25 +77,24 @@ func decodeNodeDevices(name string, str string) *GPUDevices {
 				Type:        items[3],
 				PodMap:      make(map[string]*GPUUsage),
 				Health:      health,
-				Mode:        vGPUControllerHAMICore,
 				MigTemplate: []config.Geometry{},
-				MigUsage:    config.MigInUse{},
+				MigUsage: config.MigInUse{
+					Index: -1},
 			}
-			/* 5 is the length of items before dynamic-mig feature */
-			if len(items) > 5 {
-				i.Mode = items[5]
-				if i.Mode == vGPUControllerMIG {
-					var err error
-					i.MigTemplate, err = extractGeometryFromType(i.Type)
-					if err != nil {
-						i.Mode = vGPUControllerHAMICore
-					}
+			sharingMode = getSharingMode(items[5])
```

Comment:
The length of item needs to be judged for forward compatibility.
`if len(items) > 5 {...}`